zerver/lib/export.py

```diff
@@ -1747,6 +1748,7 @@ def import_attachments(data: TableData) -> None:
     parent_id = 'attachment_id'
     child_id = 'message_id'
 
+    update_model_ids(parent_model, data, parent_db_table_name, 'attachment')
```

Comment:
Addressing the review in #8683, the change done in the commit 2602e38a4b64395cc449d77fe7b5983c7ad2246b  is not necessary (I didn't realize that the field was being deleted below)